tools/trim_galore/trim_galore.xml

```diff
@@ -48,8 +48,8 @@
         <requirement type="package" version="1.8.3">cutadapt</requirement>
         <requirement type="package" version="1.8">cutadapt</requirement>
     </requirements>
-    <version_command>
-        perl $__tool_directory/trim_galore --version
```

Comment:
I thought it was `$__tool_directory__`? Sorry if I'm missing some obvious reason it wouldn't be.